### ISSUE TITLE
EmailLink: links contain an email icon that is hidden from screen readers

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#91]
•	The purpose of the link is not clear as it only announces the person’s name.
•	[WCAG 2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)
•	Recommendation: The email icon should be surfaced to screen readers.

**Comment by McManning** [comment]
For context, this component was used in PI Portal for contact links:

![image](https://github.com/osuresearch/ui/assets/1057607/39df28c1-16b5-42b5-a694-68dad084e513)

Some additional labeling should be added for SR - e.g. "Send an email to foobar"
